source/_integrations/homekit.markdown

```diff
@@ -98,7 +98,7 @@ homekit:
       type: string
       default: '`Home Assistant Bridge`'
     ip_address:
-      description: The local network IP address. Only necessary if the default from Home Assistant does not work.
+      description: The HAP binding address. The bridge will advertise itself through the interface with this IP address, so it must be local to the machine (For Docker users, you cannot put your host IP here, because it's different from the container address and no advertisement will be generated). Only necessary if the default from Home Assistant does not work.
```

Comment:
`HAP` is not clear here and is also an abstraction that isn't used-facing.

> (For Docker users, you cannot put your host IP here, because it's different from the container address and no advertisement will be generated).

That one doesn't make sense? Home Assistant must be running on the host network?

../Frenck